SharePointFramework/ProjectExtensions/src/projectSetup/tasks/PreTask/index.ts

```diff
@@ -38,4 +40,24 @@ export class PreTask extends BaseTask {
       throw new BaseTaskError(this.taskName, strings.PreTaskErrorMessage, error)
     }
   }
+
+  /**
+   * Checks that the content types are valid and exist in the hub
+   * @param params IBaseTaskParams
+   */
+  private async validateContentTypes(params: IBaseTaskParams): Promise<void> {
+    const contentTypeToValidate: string[] = (_.toArray(params.templateSchema.Parameters) as string[])
+    .filter((item: string) => item.includes('0x'))
+
+    await Promise.all(
+      contentTypeToValidate.map(async (ct) => {
+        try {
+          await this.data.hub.web.contentTypes.getById(ct).get()
+        } catch (error) {
+          throw new BaseTaskError(this.taskName,strings.PreTaskErrorMessage,`Innholdstypen ${ct} eksisterer ikke. 
```

Comment:
Rework this to use localized strings like all other places where BaseTaskError is thrown. Example https://github.com/Puzzlepart/prosjektportalen365/blob/acdcdfd38d237b731c28caec47e97a12216efe6f/SharePointFramework/ProjectExtensions/src/projectSetup/tasks/ProvisionSiteFields/index.ts#L66